src/main/resources/application.yml

```diff
@@ -74,6 +72,16 @@ workspace:
     username: ${env.db.ws.user}
 
 terra.common:
+  kubernetes:
+    in-kubernetes: true
```

Comment:
I remember staring local WSM also read from this file? If yes, we need to fix for it.

Reply:
Hmm. There's no application-local.yml?

Reply:
I would like to get rid of using the configuration that runs a local WSM server on the laptop to run "integration" tests. It will not work for new testing, including controlled buckets and K8s disruptive tests. So I am not motivated to do the work to support it. I think we should be deploying to our personal clusters and running integration tests against those.

However, we have this limitation that the only way to run integration tests via GHA is to run the server on the GHA node. So we have to make that work - maybe by making an application-local.yml and setting the profile to include local.

Reply:
I guess people does run local server that offten. RBS has this little script: https://github.com/DataBiosphere/terra-resource-buffer/blob/master/local-dev/run_local.sh
But that is not urgent, and barely used.